Dockerfile

```diff
@@ -10,32 +10,24 @@ RUN apt-get update && apt-get install -y --no-install-recommends jq wget ca-cert
     && rm -rf /var/lib/apt/lists/*
 
 # AZCOPY INSTALL
-ARG AZCOPY_VERSION=10.27.1-20241113
-ARG user=azcopy
-ARG group=azcopy
-ARG uid=1000
-ARG gid=1000
-ARG user_home="/home/${user}"
-RUN groupadd -g ${gid} ${group} \
-    && useradd -l -d "${user_home}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"
```

Comment:
Deleting this block is incorrect: you don't create the default image user anymore.

It leads to the following warning:

```
[33m1 warning found (use docker --debug to expand):
[2025-01-13T16:50:39.508Z] [0m - UndefinedVar: Usage of undefined variable '$user' (line 70)
```

=> I believe this should be considered an error by our system (improvement for later?) but it also means we do NOT test the default user.

@smerle33 Can you re-add the user-creation related code, and we'll do a PR to improve testing later?

Reply:
I don't understand why it's not caught by this line https://github.com/jenkins-infra/docker-geoipupdate/blob/dccb23844f7b16cd9310ebe32ea3869d81c19448/cst.yaml#L4 checking the user ... ?

Reply:
same, I'm not sure why. I ran the test locally and it does not catch it. weird :|